search_events/urls.py

```diff
@@ -14,8 +14,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
```

Comment:
For multiple imports from the same package use the tuple syntax, with a trailing comma on every line. This allows easy git diffing. In this case:
```
from django.urls import (
         path,
         include,
     )
```